src/test/testUtil.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as path from 'path'
+import * as vscode from 'vscode'
+
+import * as pathutil from '../shared/utilities/pathUtils'
+
+/** Gets the full path to the project root directory. */
+export function getProjectDir(): string {
+    return path.join(__dirname, '../')
+}
+
+/** Creates a `WorkspaceFolder` for use in tests. */
+export function getWorkspaceFolder(dir: string): vscode.WorkspaceFolder {
+    const folder = {
+        uri: vscode.Uri.file(dir),
+        name: 'test-workspace-folder',
+        index: 0,
+    }
+    return folder
+}
+
+/**
+ * Asserts that filepaths are equal, after normalizing for platform differences.
+ */
+export function assertEqualPaths(actual: string, expected: string) {
```

Comment:
Does this pass or fail: `assertEqualPaths("c:\foo", "d:\foo")

Reply:
it passes, because driveletters are not always present (created a tracking issue to investigate that), so they are removed here before comparing.

If one wants strict equality then one would not use this function. Meanwhile we are testing more than ever before...

Reply:
> If one wants strict equality then one would not use this function.

What would you name such a function, if not "assertEqualPaths"?

Reply:
there is already `assert.strictEqual()` for that purpose. The lack of "strict" implies "not strict".

Reply:
There might be some confusion around this function in the future, but that is reasonable.